python/plugins/processing/tests/testdata/qgis_algorithm_tests.yaml

```
@@ -1386,6 +1386,22 @@ tests:
         name: expected/single_sided_buffer_multiline_bevel.gml
         type: vector
 
+  - algorithm: qgis:equidistancebuffer
+    name: Equidistance buffer test
+    params:
+      DISTANCE: 10000.0
+      END_CAP_STYLE: 0
+      INPUT:
+        name: points.gml|layername=points
+        type: vector
+      JOIN_STYLE: 0
+      MITER_LIMIT: 2.0
+      SEGMENTS: 8
+    results:
+      OUTPUT:
+        name: expected/equidistance_buffer_points.gml'
```

as in https://travis-ci.org/qgis/QGIS/jobs/519176767#L1542 you have a ' more at the end of filename. Not clear to me is it's enough to fix the issue
```suggestion
        name: expected/equidistance_buffer_points.gml
```